src/ripple/rpc/handlers/GetCounts.cpp

```diff
@@ -58,58 +58,50 @@ textTime(std::string& text, UptimeClock::time_point& seconds,
         text += "s";
 }
 
-// {
-//   min_count: <number>  // optional, defaults to 10
-// }
-Json::Value doGetCounts (RPC::Context& context)
+Json::Value getCountsJson(Application& app, int minObjectCount)
 {
-    int minCount = 10;
+    auto objectCounts = CountedObjects::getInstance().getCounts(minObjectCount);
 
-    if (context.params.isMember (jss::min_count))
-        minCount = context.params[jss::min_count].asUInt ();
-
-    auto objectCounts = CountedObjects::getInstance ().getCounts (minCount);
-
-    Json::Value ret (Json::objectValue);
+    Json::Value ret(Json::objectValue);
 
     for (auto const& it : objectCounts)
     {
         ret [it.first] = it.second;
     }
 
-    int dbKB = getKBUsedAll (context.app.getLedgerDB ().getSession ());
+    int dbKB = getKBUsedAll (app.getLedgerDB ().getSession ());
 
     if (dbKB > 0)
         ret[jss::dbKBTotal] = dbKB;
 
-    dbKB = getKBUsedDB (context.app.getLedgerDB ().getSession ());
+    dbKB = getKBUsedDB (app.getLedgerDB ().getSession ());
 
     if (dbKB > 0)
         ret[jss::dbKBLedger] = dbKB;
 
-    dbKB = getKBUsedDB (context.app.getTxnDB ().getSession ());
+    dbKB = getKBUsedDB (app.getTxnDB ().getSession ());
 
     if (dbKB > 0)
         ret[jss::dbKBTransaction] = dbKB;
 
     {
-        std::size_t c = context.app.getOPs().getLocalTxCount ();
+        std::size_t c = app.getOPs().getLocalTxCount ();
         if (c > 0)
```

Comment:
Not in your changeset, but there's some strangeness in this code. For exmaple, `c` is an unsigned type, why are we checking for `>0` instead of just `0`. Also, why is the declaration of `c` outside the if. I'd expect this to be written something like:
```
    if (Json::UInt c = app.getOPs().getLocalTxCount())
        ret[jss::local_txs] = c;
```
Ditto for all the `dbKB` uses above. I would have expected that to be an unsigned type and also to have it's declaration in an if statement and check for zero, not `>0`.
I wouldn't make this change in this changeset, just making a note to myself.